engine/simulationcraft.hpp

```diff
@@ -6314,10 +6316,10 @@ struct real_ppm_t
     freq( frequency ),
     modifier( mod ),
     rppm( freq * mod ),
+    first_proc_occurred( false ),
     last_trigger_attempt( timespan_t::from_seconds( -10.0 ) ),
-    last_successful_trigger( timespan_t::from_seconds( -180.0 ) ), // Blizz done lied to us, or changed it without telling. After going through a lot of logs,
-                                                                   // it seems that it's actually 3 minutes for the precombat timer.
-    initial_precombat_time( timespan_t::from_seconds( -180.0 ) ),
```

Comment:
Yes. The old model gave an absurdly high first-hit proc chance, more than 2.5x what we observe in game, while the second/third attacks and onward had very low proc rates that were significantly below what we saw in game. New testing shows this model to be much more accurate.